File: EventCheckin.ClientApp/src/app/api/models/event-member-model.ts
/* tslint:disable */
/* eslint-disable */
import { UserModel } from '../models/user-model';
export interface EventMemberModel {
  addedGuestNo?: number;
  addedParkNo?: number;
  eventId?: number;
  guestNo?: number;
  id?: number;
  isPublished?: boolean;
  parentUserId?: number | null;
  parkNo?: number;
  pendingGuestNo?: number;
  pendingParkNo?: number;
  user?: UserModel;
  userId?: number;
}

File: EventCheckin.Api/Models/Events/EventMemberModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace EventCheckin.Api.Models.Events
{
    public class EventMemberModel
    {
        public long Id { get; set; }
        public long UserId { get; set; }
        public long EventId { get; set; }
        public int GuestNo { get; set; }
        public int ParkNo { get; set; }
        public long? ParentUserId { get; set; }
        public bool IsPublished { get; set; }
        public int AddedGuestNo { get; set; }
        public int AddedParkNo { get; set; }
        public int PendingGuestNo { get; set; }
        public int PendingParkNo { get; set; }
        public UserModel User { get; set; }

        
    }
}
